index.js

```diff
@@ -373,7 +373,11 @@ class CoverageSonar extends CoverageBase {
             return getMetrics({ projectKey, startTime, endTime, prNum })
                 .then(({ coverage, tests }) => {
                     const componentId = encodeURIComponent(projectKey);
-                    const projectUrl = `${this.config.sonarHost}/dashboard?id=${componentId}`;
+                    let projectUrl = `${this.config.sonarHost}/dashboard?id=${componentId}`;
+
+                    if (projectKey.startsWith('pipeline') && prNum) {
```

Comment:
this is applicable for job scoped ones as well in enterprise mode.

Reply:
updated